packages/mongodb-chatbot-eval/src/report/reportConversationStatsForEvalRun.ts

```diff
@@ -0,0 +1,66 @@
+import { ObjectId } from "mongodb-rag-core";
+import { ReportEvalFunc, ReportEvalFuncParams } from "./ReportEvalFunc";
+import { strict as assert } from "assert";
+import { Report } from "./ReportStore";
+
+/**
+  Report on the pass percentage of the `"conversation_quality"` evaluation
+  for eval run `evaluationRunId`.
+ */
+export const reportConversationStatsForEvalRun: ReportEvalFunc = async ({
+  runId,
+  evaluationStore,
+  evaluationRunId,
+}: ReportEvalFuncParams) => {
+  const pipeline = [
+    {
+      $match: {
+        commandRunMetadataId: evaluationRunId,
+        evalName: "conversation_quality",
+      },
+    },
+    {
+      $group: {
+        _id: null,
+        totalCount: { $sum: 1 },
+        passCount: {
+          $sum: {
+            $cond: [{ $eq: ["$result", 1] }, 1, 0],
+          },
+        },
+        failCount: {
+          $sum: {
+            $cond: [{ $eq: ["$result", 0] }, 1, 0],
+          },
+        },
+      },
+    },
+    {
+      $project: {
+        _id: 0,
+        totalCount: 1,
+        passRate: {
+          $divide: ["$passCount", "$totalCount"],
+        },
```

Comment:
```suggestion
        totalCount: 1,
        passCount: 1,
        failCount: 1,
        passRate: {
          $divide: ["$passCount", "$totalCount"],
        },
```

Might be useful to include raw numbers too? We could always recreate this with a little math but the data is already here so IMO may as well include it.